core-plugins/pom.xml

```diff
@@ -239,6 +245,58 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-core_2.10</artifactId>
```

Comment:
you shouldn't add a new dependency to exclude a transitive one. If you need to exclude a transitive one, it should be excluded from the dependency that pulls it in. For example, if this is getting pulled in by cdap-unit-test, it should be excluded from there.